_data/sites/blr.json

```diff
@@ -0,0 +1,8 @@
+```json
+{
+	"url": "https://blr.design/",
+	"name": "Brian Louis Ramirez",
+	"description": "User Experience Designer based in Germany. Blogging about UX, web performance, sustainability, productivity.",
+	"twitter": "screenspan"
+}
+```
```

Comment:
This line needs to be removed to make this valid json! Thanks!

Reply:
Silly me! Of course...